app/boards/shields/tidbit/README.md

```diff
@@ -0,0 +1,68 @@
+# Building ZMK for the Tidbit
+
+Some general notes/commands for building standard Tidbit layouts from the assembly documentation.
+
+## Standard "Non Dense" Build
+
+```
+
+west build -p --board nice_nano -d build/tidbit/default -- -DSHIELD=tidbit
+
+```
+
+## Dense "19 keys" Build
+
+```
+
+west build -p --board nice_nano -d build/tidbit/19_key -- -DSHIELD=tidbit_19key
+
+```
+
+## Underglow / LEDs
+
+If you built your tidbit without the underglow leds, you'll need to add the following to one of the above commands.
+
+```
+
+-DCONFIG_ZMK_RGB_UNDERGLOW=n -DCONFIG_WS2812_STRIP=n
+
+```
```

Comment:
The standard build per the Tidbit assembly docs (https://nullbits.co/static/file/tidbit_assembly_rev01.pdf) is to include the WS2812's. However if folks leave them off the standard build I wanted to document an easy way for them to build one of the 2 keymaps with them disabled and not needing to create a full `zmk-config` if not desired.

Reply:
This way of enabling and disabling items isn't done anywhere else, so it feels out of place imo. This advice is complicated for those using a zmk-config repo. Using .conf files will work in zmk-config repos and local build environments, and it's been the standard way of doing things.

Reply:
This was only recently discovered as an option for builds. I checked with @petejohanson late 2020/12/01 / early 2020/12/02 on discord and it was unknown to be supported at the time.

I can remove these but if the below comment about removing the config vars as the LEDs are setup at the board level holds... I'd still want to add a note in the README about how to turn on the LEDs via this method if using a nice!nano which is supported (and commonly recommended in the nullbits discord [makers of the tidbit]) for builds.

Reply:
I guess what I'm mostly not sure of is how this is better than using the `.conf` files. `.conf` files are versioned and visible within user repos/build environments. This means you don't need to find the right command line arguments to make an identical build. You can list the two config names to update the RGB here.

Reply:
By having the command line args present the vast majority of Tidbit builders won't need to setup a zmk-config directory for a simple build as the assembly docs are very straight forward with regard to the underglow as well as the nullbits folks pointing to the Nice!Nano instead of any other boards.

If the majority of users aren't going to be making any changes to the default keymaps, why make them setup a zmk-config directory when all they need it for is two config options that can be enabled at build time?

Reply:
Are you expecting most people to set up a local build environment rather than using zmk-configs + github actions? Seems a bit unlikely to me since we point people towards zmk-configs when first getting started. In the case of a local build environment, I don't see what's so difficult for those users to adjust a `.conf` file.

Reply:
Can the github actions not leverage build args like -DSOMETHING=value?

Reply:
It can, but it requires messing with the build.yml, which is quite the pain for those unfamiliar compared to working with the versioned, standard `.conf` files.

Reply:
Agreed, fighting ```build.yml``` is more painful than a ```.conf```

If I add a note stating that these are examples for local development or similar verbiage would that help address the concerns?

Reply:
That would come back to the fact that I still don't see the benefits of using these command line arguments. I would personally much prefer seeing steps on how to enable these things within `.conf` files that are accessible in local development and in zmk-config repos.

Reply:
I've updated the readme to be conf file specific and remove the build args approach.